buf.yaml

```diff
@@ -1,9 +1,8 @@
-version: v1
+version: v2
 name: buf.build/bufbuild/knit
 lint:
   use:
     - DEFAULT
-  allow_comment_ignores: true
```

Comment:
Updated config version replaces this with setting `disallow_comment_ignores`, which would be false, which can be omitted from updated file.